code/modules/jobs/job_types/chaplain.dm

```diff
@@ -122,6 +119,11 @@
 	belt = /obj/item/pda/chaplain
 	ears = /obj/item/radio/headset/headset_srv
 	uniform = /obj/item/clothing/under/rank/civilian/chaplain
-	backpack_contents = list(/obj/item/camera/spooky = 1)
+	backpack_contents = list(
+		/obj/item/nullrod = 1,
+		/obj/item/choice_beacon/holy = 1,
+		/obj/item/book/granter/spell/smoke/lesser = 1,
```

Comment:
Don't give them smoke, this is map specific

Reply:
fixed